app/models/concerns/orchestration/dhcp.rb

```diff
@@ -152,10 +152,11 @@ def queue_dhcp_destroy
   end
 
   def queue_remove_dhcp_conflicts
-    return unless (dhcp? and overwrite?)
+    return if !dhcp? && !overwrite?
```

Comment:
Did you mean to change the logic here? 
`unless (dhcp? and overwrite?)` == `if !(dhcp? && overwrite?)` == `if !dhcp? || !overwrite?`